apps/client/src/features/viewers/studio/StudioClock.scss

```diff
@@ -86,6 +86,13 @@ $cyan-idle: #0aa;
       line-height: 0.8em;
     }
 
+    .studio-timer-seconds {
+      color: $red-active;
+      font-size: calc(#{$clock-size} / 4.6);
+      margin-top: calc(50% - calc(#{$clock-size} / 11));
+      line-height: 0.8em;
+    }
+
```

Comment:
If following the modifier class as proposed above, the CSS here would look something like
```
    .studio-timer {
      color: $red-active;
      font-size: calc(#{$clock-size} / 3);
      margin-top: calc(50% - calc(#{$clock-size} / 7));
      line-height: 0.8em;

      &--with-seconds {
        font-size: calc(#{$clock-size} / 4.6);
        margin-top: calc(50% - calc(#{$clock-size} / 11));
      }
    }

```